### PR TITLE
Voron0 Handle for 3mm panels

### PR DESCRIPTION
**Pull request by Zen3Dlab**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Screenshots (if appropriate)

IMG1

### CODE REVIEW

**Comment by Zen3Dhu** [comment]
Hi!

I’m a beginner at using GitHub, I don’t know if I’m doing well. :)

**Comment by Zen3Dlab** [comment]
Do I have something to do?

File: printer_mods/Zen3D/V0_Handle/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Voron0 Handle for 3mm panels
+
+## print settings:
+
+- Support required, or use Voron0Handle_supported.stl (layer hight 0.2mm)
```
Comment (KiloQubit) [inline printer_mods/Zen3D/V0_Handle/README.md:5]: Please change `hight` to `height`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thank you for your pull request.

Please fix the spelling of `height` in the README.

While the part with built-in supports may be ok, do you think it's possible to redesign the handles so that they don't require supports?  For example, maybe you can have the handle extend down instead of up and turn it upside down for printing, then reduce how far the it extends into the rail channel to something like this part https://github.com/VoronDesign/Voron-0/blob/master/VORON-0/STLs/T8_Nut_Block_16mmx22mmOD_x1.STL

File: printer_mods/Zen3D/V0_Handle/README.md (hunk 1 above)
Comment (Zen3Dlab) [reply printer_mods/Zen3D/V0_Handle/README.md]: Thanks for the suggestion! I made another version: Handle_v2.stl I apply after I try.

**Comment by KiloQubit** [comment]
Great, let me know when you test it, and thanks for making the change!

**Comment by Zen3Dlab** [comment]
> Great, let me know when you test it, and thanks for making the change!

I tried, the new version is absolutely good.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update, looks good now!

**Pull request merged**